presentation/src/main/java/com/architect/coders/mu8/utils/PresentationExtensions.kt

```diff
@@ -14,6 +19,25 @@ import androidx.lifecycle.ViewModelProvider
 import androidx.lifecycle.ViewModelProviders
 import com.bumptech.glide.Glide
 
+
+fun Context.isInternetConned(): Boolean {
+    val cm = getSystemService(Context.CONNECTIVITY_SERVICE) as ConnectivityManager
+    val activeNetwork: NetworkInfo? = cm.activeNetworkInfo
+    return activeNetwork?.isConnectedOrConnecting == true
+}
+
+fun View.makeItGone() {
+    visibility = GONE
+}
+
+fun View.makeItVisible() {
+    visibility = VISIBLE
+}
+
+fun View.makeItInvisible() {
```

Comment:
I don't think this Extension Functions are really necessary, assigning the value to the view is simpler enough already.

Reply:
it's more readable. All times we change the view's visibility

Reply:
I really see it unnecessary but let's leave it.